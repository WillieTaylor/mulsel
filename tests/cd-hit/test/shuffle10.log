================================================================
Program: CD-HIT, V4.6 (+OpenMP), Dec 11 2015, 17:28:15
Command: ./cd-hit -i shuffle10.fasta -o shuffle10.90.out -c
         0.9

Started: Fri Dec 11 18:04:27 2015
================================================================
                            Output                              
----------------------------------------------------------------
total seq: 10021
longest and shortest : 84 and 58
Total letters: 733797
Sequences have been sorted

Approximated minimal memory consumption:
Sequence        : 1M
Buffer          : 1 X 10M = 10M
Table           : 1 X 65M = 65M
Miscellaneous   : 0M
Total           : 78M

Table limit with the given memory limit:
Max number of representatives: 4000000
Max number of word counting entries: 90232192

comparing sequences from          0  to      10021
....................    10000  finished        364  clusters

    10021  finished        365  clusters

Apprixmated maximum memory consumption: 78M
writing new database
writing clustering information
program completed !

Total CPU time 0.25
================================================================
Program: CD-HIT, V4.6 (+OpenMP), Dec 11 2015, 17:28:15
Command: ./cd-hit -i shuffle10.fasta -o shuffle10.80.out -c
         0.8

Started: Fri Dec 11 18:04:27 2015
================================================================
                            Output                              
----------------------------------------------------------------
total seq: 10021
longest and shortest : 84 and 58
Total letters: 733797
Sequences have been sorted

Approximated minimal memory consumption:
Sequence        : 1M
Buffer          : 1 X 10M = 10M
Table           : 1 X 65M = 65M
Miscellaneous   : 0M
Total           : 78M

Table limit with the given memory limit:
Max number of representatives: 4000000
Max number of word counting entries: 90232192

comparing sequences from          0  to      10021
....................    10000  finished        187  clusters

    10021  finished        187  clusters

Apprixmated maximum memory consumption: 78M
writing new database
writing clustering information
program completed !

Total CPU time 0.27
================================================================
Program: CD-HIT, V4.6 (+OpenMP), Dec 11 2015, 17:28:15
Command: ./cd-hit -i shuffle10.fasta -o shuffle10.70.out -c
         0.7

Started: Fri Dec 11 18:04:28 2015
================================================================
                            Output                              
----------------------------------------------------------------
total seq: 10021
longest and shortest : 84 and 58
Total letters: 733797
Sequences have been sorted

Approximated minimal memory consumption:
Sequence        : 1M
Buffer          : 1 X 10M = 10M
Table           : 1 X 65M = 65M
Miscellaneous   : 0M
Total           : 78M

Table limit with the given memory limit:
Max number of representatives: 4000000
Max number of word counting entries: 90232192

comparing sequences from          0  to      10021
....................    10000  finished        120  clusters

    10021  finished        120  clusters

Apprixmated maximum memory consumption: 78M
writing new database
writing clustering information
program completed !

Total CPU time 0.28
================================================================
Program: CD-HIT, V4.6 (+OpenMP), Dec 11 2015, 17:28:15
Command: ./cd-hit -i shuffle10.fasta -o shuffle10.60.out -c
         0.6 -n 4

Started: Fri Dec 11 18:04:28 2015
================================================================
                            Output                              
----------------------------------------------------------------
total seq: 10021
longest and shortest : 84 and 58
Total letters: 733797
Sequences have been sorted

Approximated minimal memory consumption:
Sequence        : 1M
Buffer          : 1 X 10M = 10M
Table           : 1 X 3M = 3M
Miscellaneous   : 0M
Total           : 15M

Table limit with the given memory limit:
Max number of representatives: 4000000
Max number of word counting entries: 98011432

comparing sequences from          0  to      10021
....................    10000  finished         84  clusters

    10021  finished         84  clusters

Apprixmated maximum memory consumption: 15M
writing new database
writing clustering information
program completed !

Total CPU time 0.25
================================================================
Program: CD-HIT, V4.6 (+OpenMP), Dec 11 2015, 17:28:15
Command: ./cd-hit -i shuffle10.fasta -o shuffle10.50.out -c
         0.5 -n 3

Started: Fri Dec 11 18:04:28 2015
================================================================
                            Output                              
----------------------------------------------------------------
total seq: 10021
longest and shortest : 84 and 58
Total letters: 733797
Sequences have been sorted

Approximated minimal memory consumption:
Sequence        : 1M
Buffer          : 1 X 10M = 10M
Table           : 1 X 0M = 0M
Miscellaneous   : 0M
Total           : 12M

Table limit with the given memory limit:
Max number of representatives: 4000000
Max number of word counting entries: 98381872

comparing sequences from          0  to      10021
....................    10000  finished         63  clusters

    10021  finished         63  clusters

Apprixmated maximum memory consumption: 12M
writing new database
writing clustering information
program completed !

Total CPU time 0.25
================================================================
Program: CD-HIT, V4.6 (+OpenMP), Dec 11 2015, 17:28:15
Command: ./cd-hit -i shuffle10.fasta -o shuffle10.40.out -c
         0.4 -n 2

Started: Fri Dec 11 18:04:29 2015
================================================================
                            Output                              
----------------------------------------------------------------
total seq: 10021
longest and shortest : 84 and 58
Total letters: 733797
Sequences have been sorted

Approximated minimal memory consumption:
Sequence        : 1M
Buffer          : 1 X 10M = 10M
Table           : 1 X 0M = 0M
Miscellaneous   : 0M
Total           : 12M

Table limit with the given memory limit:
Max number of representatives: 4000000
Max number of word counting entries: 98399512

comparing sequences from          0  to      10021
....................    10000  finished         37  clusters

    10021  finished         37  clusters

Apprixmated maximum memory consumption: 12M
writing new database
writing clustering information
program completed !

Total CPU time 0.24
