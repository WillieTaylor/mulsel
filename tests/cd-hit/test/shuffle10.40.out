>seqI519 = I family seq 519  0.005806 34 I
INIQIVIIIVIEMTYIIIIPFPMVFKIIEKVIIIQTVIVEENTIIKPNIIILIIINCVVDILIRMICIIISIIGIRDVE
>seqM261 = M family seq 261  0.015076 50 M
MKPMMRPMMMMMMSCMMMEMMVMMMWWMMTSMMMMMMMMMTWQMMMCMWSPMMIMMMMMMMVMMMRMMMMMARMHQPDD
>seqS222 = S family seq 222  0.016140 55 S
FNVSESSTSSSSRSSKSSSSLSSSSSSSSSSSKWASSKSSSSSSSSQSSSSLSSSSNSSSSSSSTESSASSSSFVN
>seqY598 = Y family seq 598  0.017382 34 Y
YMERLELYYYYYYYLQAYYYYKSYYYGEQIYDYYLETAYYEYDTHHIYEETYQHHYGQGYYTYEAYYGEMRYEYYYYN
>seqE596 = E family seq 596  0.018960 36 E
DKEEEHHYDCWENGEGTEVEEAEEESEENNEREEEWEPQKDEECEETEWEHEVAETTKVYEENEEIEIEIQEELEEAKYW
>seqE265 = E family seq 265  0.018974 44 E
KCGHLEEDEEEEFEGECEENVDMEEEWEEEQEEEEEKIPEEFEEEQLTVEEEEEEEECEWETLVENEWEEEFEYEELQFK
>seqY213 = Y family seq 213  0.021769 54 Y
APNCYALYYYYYYYYTYYYYPYYPYYLYYYFYYYWYYYYYYYYYVYTYYYYYYYYYYYYYYVWYYYYYIYLPYALWKW
>seqV24 = V family seq 24  0.032825 75 V
VVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVV
>seqP472 = P family seq 472  0.054910 35 P
NLNPMFLKNPPPQPPQPPFPQFNLNWLGPSQPWGLLPYNQHTPPVHPFLPPLPQPPPTEPPPPPWPPPKPSKPPPPP
>seqR370 = R family seq 370  0.056344 35 R
IYFTDMPLRRPSTRMRARRRRDYIERKRVLRKDRRTTPGRRRIRVTIDRKRRRMQRRRRMVNRRYERRRMRRKVGLRPRQR
>seqF390 = F family seq 390  0.062097 34 F
QFPKWFFFFFFVFDFKYFYFFFFFMFLTFQLAFIQLRSNFGFMLDFFDWAWFYFTFRSFFFLFFVFKYYTIMQFFLFPQRW
>seqC359 = C family seq 359  0.067600 36 C
PGCCCACCARKCNLCACCRCFMRCCHCCACCCNMSCDCCCCCPCDCNCSCCCSCWWCNCNCAANNCLQCCTYLR
>seqA571 = A family seq 571  0.074108 36 A
FARYYAYKAYAALNNAAAAAGACLAAAAARAAFQAAEYAAIICKVAIDRAVAYACKLAQAAYSEAQNSALGRSLANLCAAAGA
>seqK88 = K family seq 88  0.082393 72 K
FKKKKKLKKKKKKKKKKKNKKKKKRKKKKKKKKKKKKKKKKKKKKKKKGKKKHKKKKKKKKKKKKKKKKKKKKKKKKK
>seqW540 = W family seq 540  0.099026 35 W
WWKWWWPQWWWWNWWFFWWSTQFLWWWLWPNNACPWWFRWKVMFWWWWPPQCWWFVWWNWDFENCWWGEWWGWFIWPM
>seqL198 = L family seq 198  0.106961 60 L
MTMTWLLLLLLLLLLLLLLLLLLLLCLGLLLLLLLLLLLELLLLLIWLLLPLLVLLKLLLDLLLLCMLKLLLLLYLLLKW
>seqR597 = R family seq 597  0.117120 35 R
VCRRFDFSRRMCYRRRRNTFVQVIRRHRARKDRMFAWGARYRWRRRRRRRAKVRRQRFATRHRVEFTRRRFRRTVRRTFPRQK
>seqP241 = P family seq 241  0.139677 46 P
DDYPPPPLPWPPPPPPPDFPPPPPRPKPPPPPPPPFPYPPPQPTMYFPPPLPKQFPPPTEPPPPYIPPPPQL
>seqF599 = F family seq 599  0.148052 35 F
DSQHFFVIFLFFTVFFNVFFNSFFMTFQFPFFLFFFFHCFFIFVIWYFQFFMFLFQFKFTEFFMQFLLFPCQFWQF
>seqG377 = G family seq 377  0.154756 35 G
FYGNGGGGGGGNWGGHTGAPGGHGGEWGWSQQGCPGGRSPGGGGGTKGGCMNNGGLRWGGRGGLAGPELKQGPFQPFQG
>seqN213 = N family seq 213  0.173363 54 N
YNENNQNNNNNNNNNNNNENNNNIRLNMENNNNPNNNNNQNNNNNNNNNCNNVENNNDNNNNSNVNNNQNQNN
>seqL445 = L family seq 445  0.177194 34 L
ETLLEFWLLLGAELLLELCLCVMCGYLVDLLYLERLSALLLQGLLLSGLLLLHLLLCLYLLYGKYLLLKP
>seqD573 = D family seq 573  0.222011 36 D
MDDDDMSASDDTDDIEDLRCDDIQMQESVDDEDTTCLDTSQDDDDIDDDQYDAPADQDEDMDHDMDKIDDQDDIDDD
>seqQ578 = Q family seq 578  0.227982 34 Q
LQGQRRQVPQQQELNLQQQQKGDRQQQQQQVDQGAQQWQIQQEQVMQDVQQFAGQQTNQQCQQRQDRTVQPGY
>seqS577 = S family seq 577  0.233759 34 S
SSRFSYSSSESNSFGFNRFGNSTLMSVSSITSKSMSGSSFSSSSKSTVSMYSKDMGGGSSLSSLSSTASISSSN
>seqW295 = W family seq 295  0.234465 35 W
IPPLWWCHEWWWQLKGWWIWWWWWKWHWWAFAWWWHWCWTWWWIWPWRWWAWWMHWAPWACYWWWWHCIR
>seqI286 = I family seq 286  0.267230 39 I
NCVVIRMMIIEIIINIRILISGAVWIIIIVMIIEPIIIIIIWIIIIIMIIIINQIICILIMINIARIIIGFA
>seqD172 = D family seq 172  0.284442 61 D
NIWDDDDDDDDDDDKDDDDIDTWDDDDDDDDDDDDCDDDDDTDDTDDDCDDDDDDDDDDDDDDDDDDDDDDDKY
>seqM486 = M family seq 486  0.312235 34 M
QMYMCMQQDYEMMWMMWMMNKIGRSMPHMMWWAIISFMMMMMMHWSPVMMNMMSLARKDHWMMMMSMMMCYMMMQRMWEHQTNM
>seqT505 = T family seq 505  0.396160 34 T
MTMEFTFTLTTRTSDTAESGTTTTDPTCWTTDTYTTMCHTTNSCTTLMFTTTTKTPHTTHELTMTNTKHRTYTYCTMHFKIG
>seqH475 = H family seq 475  0.416910 35 H
HPHRHWNHWHLHKHHHLQHWTHCAHHIHHHHHWHDIWPHTHHACHHHFHIAHAAMGHHHNGRAMHHHNAHLTIRTKAHK
>seqQ353 = Q family seq 353  0.642107 34 Q
QHVGQQQVTSQELHHVYMQQQVDAFCRYQQSQLQQQNQASARQQQKQKMQDSQLEQMQADFQHQQSQQQQQEQHRTQQPL
>seqA426 = A family seq 426  0.665350 34 A
AQSAKYANKAAAAAAAAVAFLAACSAQAAAVGAAAAASVCMAANYMAACIKQGIAAAMAECAVAGRSSMSTAVIGMT
>seqS394 = S family seq 394  0.739591 34 S
FRSNSVVELSIGSNSSSSSSMMSSLSFRDSWQSRMSSKFILKGSKSSSLSSYQYSGSLDMIGNAGSSLGSSSVSCASSCYASVN
>seqN507 = N family seq 507  0.745134 35 N
NHNNHRNNNCNDQNPNNNYPNNIETNANQNWHQHNENIVANNNNSQNIGFQENEMNNRHNVFNDNNSNNFASYNILSNMN
>seqG594 = G family seq 594  0.824516 34 G
GEGYGGGGGGTGLGYWPEGGSHPGGKPGSSHGGGGGCGKNKPTGSLFGGGHGPGHGGGGGYGEFG
>seqK590 = K family seq 590  0.951817 34 K
MIKFKYKVWKGVFKKVPKKKEVVKKKKKKEKYKKVPKSKKDKNMKTNHVWQLSNHQKKSDQDTDTKQVNNDKCKKKKINYKKK
